include/ck/wrapper/tensor.hpp

```diff
@@ -12,25 +12,26 @@ namespace wrapper {
 
 /**
  * \brief Tensor wrapper that performs static and dynamic buffer logic.
+ * The tensor is based on a descriptor stored in the Layout. Additionally,
+ * tensor can be sliced or shifted using multi index offset.
  *
  * \tparam BufferAddressSpace Memory type (Generic, Global, LDS, VGPR, SGPR).
  * \tparam ElementType Element data type.
  * \tparam Shape Tensor shape (layout component).
- * \tparam UnnestedDescriptorType Unnested descriptor (layout component).
- * \tparam NumVectors Number of vectors (only for VGPR, SGPR).
- * \tparam ScalarPerVector Scalars per vector (only for VGPR, SGPR).
+ * \tparam UnnestedDescriptorType Flatten descriptor (layout component).
```

Comment:
UnnestedDescriptorType  -> `FlatDesc...`
```suggestion
 * \tparam UnnestedDescriptorType Flat tensor descriptor type (layout component).
```